tests/unit/cmd/honeytoken/test_honeytoken.py

```diff
@@ -143,3 +144,51 @@ def payload_checker(body: Union[List[str], Dict[str, Any]]) -> None:
     assert_invoke_exited_with(result, ExitCode.UNEXPECTED_ERROR)
     mock.assert_all_requests_happened()
     assert "ggshield does not have permissions to create honeytokens" in result.stdout
+
+
+@pytest.mark.parametrize(
+    "subcommand, endpoint",
+    [
+        ("create", "/honeytokens"),
+        ("create-with-context", "/honeytokens/with-context"),
+    ],
+)
+def test_honeytoken_create_error_403_allowlist(
```

Comment:
That's a lot of duplicated code with the previous test, couldn't we mutualize and parametrize all scenariis in one test ? 🤔

Reply:
Yes, I factorized tests